pkg/testcase/upgradecluster.go

```diff
@@ -35,7 +35,15 @@ func TestUpgradeClusterSUC(version string) error {
 		return err
 	}
 
-	originalFilePath := shared.BasePath() + "/distros-test-framework/workloads/amd64/rke2-upgrade-plan.yaml"
+	product, err := shared.GetProduct()
+	Expect(err).NotTo(HaveOccurred())
+
+	var originalFilePath string
+	originalFilePath = shared.BasePath() + "/distros-test-framework/workloads/amd64/rke2-upgrade-plan.yaml"
```

Comment:
Suggestion: As we are getting the product, we can just have one liner and remove the conditional
```
originalFilePath := shared.BasePath() + fmt.Sprintf("/distros-test-framework/workloads/amd64/%s-upgrade-plan.yaml", product)
```

Reply:
Done, thanks for the suggestion!